### DISCUSSION TITLE
Hiding tables on app.supabase.io web ui

### DISCUSSION THREAD

**Discussion by rphly** [supabase/supabase#1965] [Questions] [answered]
I have a table that I would like to keep private from other members of the supabase org. How may I go about doing this?

**Comment by silentworks** [comment, marked as answer]
You could add this to another schema as only the `public` schema is shown inside of the web UI. The database currently has an `auth`, `storage` and some other schemas which don't show in the UI, you could create your own and it won't show up in the UI.

Don't add my new tables to any of the existing schemas that are not `public` unless it's your own that you have created.

You will probably need to add the new schema to the **Extra search path** field under `settings/api` in order for the js library to pick up the tables properly.

![Screenshot 2021-06-15 154058](https://user-images.githubusercontent.com/79497/122083166-24500080-cdf0-11eb-8aa7-434814aaab3c.png)
